data/json/monster_factions.json

```diff
@@ -123,5 +123,20 @@
         "type" : "MONSTER_FACTION",
         "name" : "spider_cellar",
         "base_faction" : "spider"
+    },{
+        "type" : "MONSTER_FACTION",
+        "name" : "mutant",
+        "neutral" : ["vermin"],
+        "by_mood" : ["small_animal", "blob", "utility_bot", "mutant_omnivore"]
+    },{
+        "type" : "MONSTER_FACTION",
+        "name" : "mutant_omnivore",
```

Comment:
Mutants should have "mutant" faction as a base faction, so that other factions can reference that without having to name every single sub-faction.